include/alpaka/queue/QueueHipRtBlocking.hpp

```diff
@@ -52,11 +52,11 @@ namespace alpaka
             {
                 //#############################################################################
                 //! The HIP RT blocking queue implementation.
-                class QueueHipRtBlockingImpl final
+                class QueueHipRtBlocking final
```

Comment:
Please do not remove the `Impl` postfix. It is an error that it is missing from `QueueCudaRtBlocking`.
You should probably add it there instead of removing it here.

Reply:
ok, no prob, I'll do that.